local/location.go

```diff
@@ -104,7 +123,16 @@ func (l *location) Container(id string) (stow.Container, error) {
 	if !ok {
 		return nil, errors.New("missing " + ConfigKeyPath + " configuration")
 	}
-	containers, err := l.filesToContainers(path, id)
+	var fullPath string
+	if filepath.IsAbs(id) {
+		//Absolute path - id has native OS semantics
```

Comment:
comments aren't adding any value in the if/else block here, obvious from the code itself

Reply:
axe em